### PR TITLE
disable revolutionaries

### PR DESCRIPTION
**Pull request by CrigCrag**
## About the PR

this pr disables revolutionaries
## Why / Balance

keronshb suggested it would be a good idea because people keep making bandaid PRs instead of fixing larger problems with the gamemode.
## Technical details

removes the possibility for revs to be selected in secret

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- remove: Removed the revolutionaries gamemode pending a rework.

### CODE REVIEW

**Comment by CrigCrag** [comment]
i cant wait to get a free pipe bomb in my mailbox

File: Resources/Prototypes/secret_weights.yml [hunk 1]
```diff
@@ -1,8 +1,7 @@
 - type: weightedRandom
   id: Secret
   weights:
-    Nukeops: 0.20
-    Traitor: 0.50
+    Nukeops: 0.30
+    Traitor: 0.60
     Zombie: 0.10
-    Revolutionary: 0.20
```
Comment (metalgearsloth) [inline Resources/Prototypes/secret_weights.yml:7]: How to tell someone doesn't like zombie.

Comment (deltanedas) [reply Resources/Prototypes/secret_weights.yml:7]: good

Comment (CrigCrag) [reply Resources/Prototypes/secret_weights.yml:7]: add peashooters then we raise the probability

**Comment by lzk228** [comment]
maybe we can bring back survival with very small chance? because only 3 gamerules its a too few

**Comment by keronshb** [comment]
> maybe we can bring back survival with very small chance? because only 3 gamerules its a too few

I'm trying to get most magic stuff done this week so we can have wizard in the next couple of weeks or so.

**Comment by lzk228** [comment]
> I'm trying to get most magic stuff done this week so we can have wizard in the next couple of weeks or so.

Soon™ :trollface:

**Comment by CrigCrag** [comment]
when the test fail is fake

**Comment by metalgearsloth** [comment]
It's not a fake test fail, your master is old so update it.

**Comment by CrigCrag** [comment]
got dammit

**Comment by Stealthbomber16** [comment]
It was good while it lasted.

**Comment by keronshb** [comment]
Update your master and this is good to go

**Comment by CrigCrag** [comment]
i fetched and merged and am not quite sure how to get it to show up on this branch without breaking everything

**Comment by keronshb** [comment]
> i fetched and merged and am not quite sure how to get it to show up on this branch without breaking everything

Pull master into this branch, then push.

**Comment by CrigCrag** [comment]
win?

**Comment by KittenColony** [comment]
pogging?

**Comment by coolmankid12345** [comment]
It's so joever

**Comment by CrigCrag** [comment]
image

**Comment by CrigCrag** [comment]
test passed pro

**Comment by MACMAN2003** [comment]
THE TYRANNY OF THE REVOLUTION IS temporarily GONE!!

**Comment by K-Dynamic** [comment]
I'm surprised how uncontroversial this was lol

**Comment by drongood12** [comment]
Sometimes later... 
The game was deleted

**Comment by FairlySadPanda** [comment]
Great now I have to actually do my Revs refactor

**Pull request merged**